src/commands/deploy.ts

```diff
@@ -0,0 +1,279 @@
+import boxen from "boxen";
+import chalk from "chalk";
+import ora from "ora";
+import type { ArgsSpec } from "src/services/command/arg.js";
+import { AppArg } from "../../src/services/app/arg.js";
+import { REMOTE_SERVER_CONTRACT_STATUS_SUBSCRIPTION } from "../../src/services/app/edit-graphql.js";
+import { type Command, type Usage } from "../../src/services/command/command.js";
+import { config } from "../../src/services/config/config.js";
+import { FileSync } from "../../src/services/filesync/filesync.js";
+import { isEqualHashes } from "../../src/services/filesync/hashes.js";
+import { select } from "../../src/services/output/prompt.js";
+import { sprint } from "../../src/services/output/sprint.js";
+import { isGraphQLErrors } from "../../src/services/util/is.js";
```

Comment:
I ran into this error while testing out this command on the new experimental version:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/opt/homebrew/lib/node_modules/ggt/src/services/app/arg.js' imported from /opt/homebrew/lib/node_modules/ggt/lib/commands/deploy.js
    at finalizeResolution (node:internal/modules/esm/resolve:264:11)
    at moduleResolve (node:internal/modules/esm/resolve:917:10)
    at defaultResolve (node:internal/modules/esm/resolve:1130:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///opt/homebrew/lib/node_modules/ggt/src/services/app/arg.js'
}
```

That error is happening because the `src` directory doesn't exist when we ship ggt -- all the `.ts` files get compiled to `.js` files inside the `lib` directory.

We can fix this by changing the imports from this:

```ts
import { ... } from "../../src/services/command/arg.js"
```

To this:

```ts
import { ... } from "../services/command/arg.js"
```

```suggestion
import type { ArgsSpec } from "../services/command/arg.js";
import { AppArg } from "../services/app/arg.js";
import { REMOTE_SERVER_CONTRACT_STATUS_SUBSCRIPTION } from "../services/app/edit-graphql.js";
import { type Command, type Usage } from "../services/command/command.js";
import { config } from "../services/config/config.js";
import { FileSync } from "../services/filesync/filesync.js";
import { isEqualHashes } from "../services/filesync/hashes.js";
import { select } from "../services/output/prompt.js";
import { sprint } from "../services/output/sprint.js";
import { isGraphQLErrors } from "../services/util/is.js";
```